lib/rgeo/geo_json/conversion_methods.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module RGeo
+  # This module serves to provide handy methods when using GeoJSON. The methods
+  # provided eases the passage between entities and GeoJSON String/Hash.
+  module GeoJSON::ConversionMethods
+    # Convert a geometry to a GeoJSON Hash
+    def as_geojson
+      GeoJSON.encode(self)
+    end
+    alias as_json as_geojson
+
+    # Convert a geometry to a GeoJSON String
+    def to_geojson
+      ::MultiJson.dump(as_geojson)
+    end
+    alias to_json to_geojson
+  end
+
+  module Feature
+    module Instance
```

Comment:
Could we just `include GeoJSON::ConversionMethods` here? It seems like this is identical to that module. Or does that not work because of where the modules are defined?

Reply:
We are unable to do this as after rgeo core is required, the `Feature::Instance` module has already been included into the various geometry implementations. By including another module in `Feature::Instance` after it has been inserted into the ancestor tree, the latter will not be automatically included into all the other modules which have included `Feature::Instance`. There are patches which could make it happen, but duplicating the code is the easiest solution.

Reply:
@teckwan I guess we could afford a technical comment in the code to explain that duplication?

Reply:
Done!

Reply:
Very helpful description thanks